packages/flame/lib/src/game/game_loop.dart

```diff
@@ -28,6 +28,10 @@ class GameLoop {
     _ticker.stop();
   }
 
+  void dispose() {
+    _ticker.dispose();
+  }
```

Comment:
Added access to dispose via our gameloop. The actual implementation of both tickers stop and dispose are quite similar in terms of freeing resources, but this is a matter of internal flutter implementation, we should use the public API as intended since internal stuff can change there.